Sources/Brave/Frontend/Player/Player.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<!-- Copyright 2023 The Brave Authors. All rights reserved.
+  -- This Source Code Form is subject to the terms of the Mozilla Public
+  -- License, v. 2.0. If a copy of the MPL was not distributed with this
+  -- file, You can obtain one at https://mozilla.org/MPL/2.0/.
+  -->
+<html lang="en">
+
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, viewport-fit=cover">
```

Comment:
can we add a meta content-security-policy tag to lock this down a bit? https://content-security-policy.com/examples/meta/

as a starting policy i suggest `default-src 'self'; frame-ancestors 'none'; base-uri 'none'; form-action 'none'; child-src 'none';object-src 'none'; frame-src https://www.youtube-nocookie.com`

Reply:
Doesn't seem to be a problem adding these